pkgs/applications/misc/tippecanoe/default.nix

```diff
@@ -19,12 +19,16 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
   doCheck = true;
 
+  passthru.tests.version = testers.testVersion {
```

Comment:
More reliable would be to self-reference using the `finalAttrs.finalPackage` mechanism https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes